tests/e2e/configurer/chain/commands.go

```diff
@@ -302,3 +303,17 @@ func (n *NodeConfig) QueryPropStatusTimed(proposalNumber int, desiredStatus stri
 	elapsed := time.Since(start)
 	totalTime <- elapsed
 }
+
+func (n *NodeConfig) Status() (any, error) {
+	cmd := []string{"osmosisd", "status"}
+	_, errBuf, err := n.containerManager.ExecCmd(n.t, n.Name, cmd, "")
+	if err != nil {
+		return nil, err
+	}
+	var result any
+	err = json.Unmarshal(errBuf.Bytes(), &result)
+	if err != nil {
+		return nil, err
+	}
+	return result, nil
```

Comment:
I'm not sure if this parsing right. What is the final structure / format of the `result`? How can we be sure that it parses correctly?

Reply:
its a json stringify

Reply:
Got some problems when converting `string` to golang native types

Reply:
Got some problems when converting `string` to golang native types

Reply:
What does the problem / error say?

Reply:
At what commit exactly were you seeing this error?

Reply:
https://github.com/osmosis-labs/osmosis/pull/4150/commits/af9decbe9171a3f69407878bb89c053a5e6d9e0d

Reply:
its fixed, sr for take it long time